src/lib/utils/asyncStorage.js

```diff
@@ -0,0 +1,48 @@
+import AsyncStorage from '@react-native-community/async-storage'
+import { isFunction } from 'lodash'
+
+export default new class {
```

Comment:
it would be good to have some unit tests for that class (if in possible to do in the jest. probably there're some ways to mock window.localStorage).

it it could be done quickly (during today) - please do that. add a few cases storing / reading values of the different types (e.g. strings numbers, bools, null/undefined, objects, arrays of the different items types)